resources/css/south-african-covid-19-page-banner.css

```diff
@@ -0,0 +1,115 @@
+body.covid_top_scroll,
```

Comment:
So the only thing here is the WP coding standards for PHP and CSS. Check them out here:

https://make.wordpress.org/core/handbook/best-practices/coding-standards/css/
https://make.wordpress.org/core/handbook/best-practices/coding-standards/php/

I'll try drop in some methods to automate the testing of this.

Reply:
I have made some changes which I think should be fine now.